app_description.txt

```diff
@@ -0,0 +1,49 @@
+Zingo! A Zcash Application
+
+Use zero knowledge cryptography to manage your transaction history in a private and mindful way.
+
+Take exclusive and unique control of your financial history. Use this unique insight into your
+own financial behavior, to introspect on how you spend money. Share insights on your terms. 
+Establish healthy relationships with diverse economic communities.
+
+Only you have the power to share your information! Your perspective matters. You matter.
+
+When you send Zcash you can write a memo containing anything you like. When you receive
+Zcash you also receive the memo that the Sender wrote. As with most cryptocurrencies, wallets
+routinely send transactions to themselves to recover "change" from the transaction.  In this case, where
+the application is both the Sender and the Receiver, Zingo uses the memo system to write
+notes to itself about the transactions it builds for the User.
+
+Zingo extracts that data to provide insights exclusively to you.  In our initial release we
+already use this introspective mechanism to "remember" which addresses you've previously sent Zcash.
```

Comment:
This already exists by-default for the sapling and transparent pools, so I'm worried that this sounds like we're making a mountain out of something that someone not familiar with the UA problem would think "My wallet already does that". Maybe "which full unified addresses you've previously sent Zcash" or something along those lines?